### DISCUSSION TITLE
Cant view webcams in homeassistant connected through TOR(No stealth mode)

### DISCUSSION THREAD

**Discussion by chaitanya-smartrotamac** [hassio-addons/app-tor#113] [Q&A]
Cant view webcams in home assistant connected through TOR(No stealth mode)
I have Ip camera added to HA through iframe but that frame is not viewable through the browser in remote why?

**Comment by foo-calls-bar** [comment]
> Cant view webcams in home assistant connected through TOR(No stealth mode)
> I have Ip camera added to HA through iframe but that frame is not viewable through the browser in remote why?

Can you at least login to HA at your onion address and view your dashboard?

**Reply by chaitanya-smartrotamac** [reply]
Yes I can login.
I can see dashboard.

But fileeditor addon shows : unauthorised
Why?

Next
I have a iframe configured as
type: iframe
url: http://192.168.1.11
aspect_ratio: 100%
Actually that is my webcam configured as webpage.

I can't see it why?

**Reply by chaitanya-smartrotamac** [reply]
My tor configuration


socks: false
hidden_services: true
stealth: false
client_names: []
ports:
  - 192.168.xx.xx:8123:8080
log_level: info


I tried 

ports:
  - 192.168.xx.xx:8123:8080
  - 81
 
ports:
  - 192.168.xx.xx:8123:8080
  - 192.168.xx.xx:81


 ports:
  - 192.168.xx.xx:8123:8080
  - 192.168.xx.xx:81/view

**Reply by chaitanya-smartrotamac** [reply]
Info log data


Add-on: Tor
 Protect your privacy and access Home Assistant via Tor.
-----------------------------------------------------------
 Add-on version: 3.0.5
 You are running the latest version of this add-on.
 System: Home Assistant OS 6.6  (aarch64 / raspberrypi3-64)
 Home Assistant Core: 2021.10.7
 Home Assistant Supervisor: 2021.10.8
-----------------------------------------------------------

**Reply by chaitanya-smartrotamac** [reply]
Any info required?

**Reply by chaitanya-smartrotamac** [reply]
My requirement :
Home assistant working
Webcam connected with IP
Tor give me private access to HA
I need to see webcam feed

**Reply by foo-calls-bar** [reply]
So you are trying to access the web cam directly over TOR? Is that right? What version of TOR are you using on the client side browser? Because V2 onion addresses are no longer reachable in the latest TOR release. In order to migrate to a V3 address now you'll have to pull the main branch of this repo (which has already been patched accordingly) and install it as a local HA add-on until the next official addon-tor release. But only if you aren't using stealth mode. That patch has been submitted but is still awaiting approval from this repo's owner.

**Reply by chaitanya-smartrotamac** [reply]
Can we access the webcam directly over Tor?
How??any tutorial?

I am just using a webpage card in overview of home assistant to show my IP camera in HA.

It is showing in local.
It is shown in Home assistant in local network also.
But in Tor can't see it.

**Reply by chaitanya-smartrotamac** [reply]
It is a http site is that a issue??

**Reply by chaitanya-smartrotamac** [reply]
Finally I made my webcams working by going to ESPHOME.

Now tor itself is not accessible.
Is TOR down?
